test/unit/test_sap_http_oauth.py

```
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3
+
+import json
```

Flake8 (7.3.0)

[error] 'json' imported but unused

(F401)